python/tests/test_serialization.py

```diff
@@ -0,0 +1,682 @@
+from dataclasses import dataclass
+import pytest
+from scylla.session_builder import SessionBuilder
+from scylla._rust.session import Session  # pyright: ignore[reportMissingModuleSource]
+import pytest_asyncio
+from typing import Callable, Awaitable, AsyncGenerator
+from decimal import Decimal
+from datetime import datetime, date, time, timezone
+from dateutil.relativedelta import relativedelta
+import ipaddress
+import uuid
+from dataclasses import asdict
+from typing import List, Tuple, Optional
+
+
+async def set_up() -> Session:
+    builder = SessionBuilder(["127.0.0.2"], 9042)
+    session = await builder.connect()
+
+    await session.execute("""
+            CREATE KEYSPACE IF NOT EXISTS testks
+            WITH replication = {'class': 'SimpleStrategy', 'replication_factor': 1};
+        """)
```

Comment:
:wrench: Please don't use `SimpleStrategy`.
```suggestion
    await session.execute("""
            CREATE KEYSPACE IF NOT EXISTS testks
            WITH replication = {'class': 'NetworkTopologyStrategy', 'replication_factor': 1};
        """)
```